crates/alkahest-renderer/src/renderer/mod.rs

```diff
@@ -112,7 +119,7 @@ impl Renderer {
             self.draw_shading_pass(scene);
             self.draw_transparents_pass(scene);
 
-            draw_utilities(self, scene, resources)
+            draw_utilities(self, scene, resources, current_hash)
```

Comment:
Map hash should **not** be passed into the renderer.
Is there a way we can identify local nodes without needing information about maps (besides the scene)?
If not, then we might need to pass a `Map` into the renderer instead of a `Scene`